vsts/node-release.yaml

```diff
@@ -0,0 +1,185 @@
+name: Release_$(BuildID)_$(Date:yyyyMMdd)$(Rev:.r)
+resources:
+- repo: self
+  clean: true
+
+phases:
+
+- phase: Phase_1
+  displayName: Create Release Branch
+
+  condition: succeededOrFailed()
+  queue:
+    name: Hosted
+
+  steps:
+
+  - checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+    persistCredentials: true  # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+
+  - task: NodeTool@0
+    displayName: 'Use Node 6.x'
+    enabled: false
```

Comment:
>false [](start = 13, length = 5)

if the task is disabled it should be removed. if it's enable, it should probably be node 8 or 10, 6 is kindof old now.